octoprint_v8theme/static/css/main.css

```diff
@@ -1,52 +1,1108 @@
-#temperature-graph {
-    background-image: none !important;
-    height: 150px;
-}
+/* Voxel8 OctoPrint Theme
+ * Contributors: Jack Minardi (jack@voxel8.co), Kevin Murphy (kevin@voxel8.co)
+===================================================  
+   Table of Contents
+===================================================  
+    I. SHARED STYLES
+   II. PRIMARY COMPONENTS
+  III. PANELS
+   IV. THEME ELEMENTS
+    V. COLOR SYSTEM
+   VI. MISCELLANEOUS
+  VII. HELPERS
+ VIII. RESPONSIVE FUNCTIONALITY
+===================================================
 
-.legend {
-    display: none;
+===================================================
+  I. SHARED STYLES
+	 A. Global
+	 B. Grid
+	 C. Typography
+===================================================
+  A. Global
+=================================================== */
+@import url('http://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800');
```

Comment:
We were using custom fonts offline before.  You should be able to save whatever gets downloaded from Google.  I would definitely test that everything works and looks good offline.